<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buttongroup</class>
 <widget class="QWidget" name="buttongroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="user_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>游客</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/res/title_user.png</normaloff>:/res/title_user.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="myfile_btn">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>我的文件</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/res/title_file.png</normaloff>:/res/title_file.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="share_btn">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;
</string>
     </property>
     <property name="text">
      <string>共享列表</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/res/title_share.png</normaloff>:/res/title_share.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tran_btn">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>传输列表</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/res/title_data.png</normaloff>:/res/title_data.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="switch_btn">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>75</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>切换用户</string>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/res/title_change.png</normaloff>:/res/title_change.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="popupMode">
      <enum>QToolButton::DelayedPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>232</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="hide_but1">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/res/title_min.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="expend_but">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/res/title_max.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="close_but_1">
          <property name="styleSheet">
           <string notr="true">border-image: url(:/res/title_close.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
